src/server/socketHandlers/scUnsubscribeHandler.js

```diff
@@ -0,0 +1,11 @@
+import parseChannel from './parseChannel';
+import {LEAVE, PRESENCE} from 'universal/subscriptions/constants';
+
+export default function scUnsubscribeHandler(exchange, socket) {
+  return async function unsubscribeHandler(subbedChannelName) {
+    const {channel} = parseChannel(subbedChannelName);
+    if (channel === PRESENCE) {
+      exchange.publish(subbedChannelName, {type: LEAVE, socketId: socket.id});
```

Comment:
I'm really happy with how this turned out. Pretty dern expressive & talk about simple!